HPXMLtoOpenStudio/measure.rb

```diff
@@ -4169,15 +4191,102 @@ def self.set_surface_exterior(model, spaces, surface, exterior_adjacent_to)
       surface.createAdjacentSurface(create_or_get_space(model, spaces, HPXML::LocationLivingSpace))
       @cond_bsmnt_surfaces << surface
+    elsif [HPXML::LocationOtherHeatedSpace, HPXML::LocationOtherMultifamilyBufferSpace, HPXML::LocationOtherNonFreezingSpace].include? exterior_adjacent_to
+      add_otherside_coefficients(surface, exterior_adjacent_to, model, spaces)
+      surface.setSunExposure('NoSun')
+      surface.setWindExposure('NoWind')
     else
       surface.createAdjacentSurface(create_or_get_space(model, spaces, exterior_adjacent_to))
     end
   end
 
+  def self.add_otherside_coefficients(surface, exterior_adjacent_to, model, spaces)
+    if spaces[exterior_adjacent_to].nil?
+      # Create E+ other side coefficient object
+      otherside_object = OpenStudio::Model::SurfacePropertyOtherSideCoefficients.new(model)
+      otherside_object.setName(exterior_adjacent_to)
+      # Fixme: assumption the same as SurfacePropertyConvectionCoefficients of return air plenum
+      otherside_object.setCombinedConvectiveRadiativeFilmCoefficient(30)
+      # Schedule of space temperature, can be shared with water heater/appliances
+      create_outside_boundary_schedule(model, exterior_adjacent_to, spaces)
+      # FIXME: wait for new OS release with bugfix of https://github.com/NREL/OpenStudio/issues/3848
+      otherside_object.setPointer(9, @mf_temp_sch_map[exterior_adjacent_to].handle)
+      surface.setSurfacePropertyOtherSideCoefficients(otherside_object)
+      spaces[exterior_adjacent_to] = otherside_object
+    else
+      surface.setSurfacePropertyOtherSideCoefficients(spaces[exterior_adjacent_to])
+    end
+  end
+
+  def self.create_outside_boundary_schedule(model, outside_space, spaces)
+    # Create outside boundary schedules to be actuated by EMS,
+    # can be shared by any surface, duct, or appliances adjacent to / located in those spaces
+
+    # return if already exists
+    return if not @mf_temp_sch_map[outside_space].nil?
+
+    @mf_temp_sch_map[outside_space] = OpenStudio::Model::ScheduleConstant.new(model)
+    @mf_temp_sch_map[outside_space].setName("#{outside_space}")
+
+    if outside_space == HPXML::LocationOtherHeatedSpace
+      # Average of indoor/outdoor temperatures with minimum of 68 deg-F
+      temp_min = UnitConversions.convert(68, 'F', 'C')
+      indoor_weight = 0.5
+      outdoor_weight = 0.5
+    elsif outside_space == HPXML::LocationOtherMultifamilyBufferSpace
+      # Average of indoor/outdoor temperatures with minimum of 50 deg-F
+      temp_min = UnitConversions.convert(50, 'F', 'C')
+      indoor_weight = 0.5
+      outdoor_weight = 0.5
+    elsif outside_space == HPXML::LocationOtherNonFreezingSpace
+      # Floating with outdoor air temperature with minimum of 40 deg-F
+      temp_min = UnitConversions.convert(40, 'F', 'C')
+      indoor_weight = 0.0
+      outdoor_weight = 1.0
+    elsif outside_space == HPXML::LocationOtherHousingUnit
+      # For water heater, duct, appliances, etc.
```

Comment:
The comments here and elsewhere suggest that this applies to appliances, but I don't think that's true. Yes, ducts and water heaters will point to these space types, but appliances won't.